conf/tests/germline_sv.config

```diff
@@ -40,6 +40,6 @@ params {
     include_expression   = 'FILTER="."'
 
     truth_vcf            = "${params.test_data_base}/testdata/hg37/truth/germline/HG002_SVs_Tier1_v0.6.chr21.vcf.gz"
-    regions_bed          = "${params.test_data_base}/testdata/hg37/truth/germline/HG002_SVs_Tier1_v0.6.chr21.bed"
+    targets_bed          = "${params.test_data_base}/testdata/hg37/truth/germline/HG002_SVs_Tier1_v0.6.chr21.bed"
```

Comment:
Can you highlight the name change of this parameter a bit more in the changelog?